python-sdk/examples/opencv/README.md

```diff
@@ -0,0 +1,17 @@
+# Distributed Image Processing with OpenCV and Python Pachyderm
+
+This is a reproduction of Pachyderm's opencv example in python. A walkthrough is available in the Pachyderm *[docs](https://docs.pachyderm.io/en/latest/getting_started/beginner_tutorial.html)*
+
+This example showcases the pachyderm-sdk analogs of common `pachctl` commands, such as creating repos and pipelines or getting a file.
+
+**Prerequisites:**
+- A workspace on [Pachyderm Hub](https://docs.pachyderm.com/latest/hub/hub_getting_started/) (recommended) or Pachyderm running [locally](https://docs.pachyderm.com/latest/getting_started/local_installation/)
+- pachyderm-sdk installed. A pip package is coming soon, but for now, you can run from the python-sdk directory:```
```

Comment:
I think you can leave out the  "pip package coming soon" since it's likely to be forgotten when this gets release, and say "install the latest package or ..."